api/handler/trigger.go

```diff
@@ -41,7 +41,7 @@ func trigger(router chi.Router) {
 //	@param		validate	query		bool									false	"For validating targets"
 //	@param		body		body		dto.Trigger								true	"Trigger data"
 //	@success	200			{object}	dto.SaveTriggerResponse					"Updated trigger"
-//	@failure	400			{object}	api.ErrorInvalidRequestExample			"Bad request from client"
+//	@failure	400			{object}	interface{}								"Bad request from client"
```

Comment:
```suggestion
//	@failure	400			{object}	interface{}								"Bad request from client. Could return: dto1, dto2"
```
Предлагаю добавить в коммент какие dto-шки могут быть возвращены при 400. (Чтобы совсем уж пользователей в неведении не держать)

Reply:
Сделал